BackgroundChanger.js

```diff
@@ -0,0 +1,31 @@
+const TIME = 10;
+
+function generateColors() {
+  const colors = {};
+  const randomColor = () => {
```

Comment:
I had this nested because it was relevant to this function. However, I think it would be fine to not have it nested as well.